lib/client/sse_client.dart

```diff
@@ -35,19 +35,23 @@ class SseClient extends StreamChannelMixin<String?> {
 
   int _lastMessageId = -1;
 
+  late String _clientId;
+
   late EventSource _eventSource;
 
   late String _serverUrl;
 
   Timer? _errorTimer;
 
   /// [serverUrl] is the URL under which the server is listening for
-  /// incoming bi-directional SSE connections.
-  SseClient(String serverUrl) {
-    var clientId = generateUuidV4();
+  /// incoming bi-directional SSE connections. [debugKey] is an optional key
+  /// that can be used to identify the SSE connection.
+  SseClient(String serverUrl, {String? debugKey}) {
+    var uuid = generateUuidV4();
+    _clientId = debugKey == null ? uuid : '$debugKey-$uuid';
```

Comment:
I think we could avoid `late` if we put the `generateUuidV4()` call in each branch

```dart
final String _clientId;

SseClient(String serverUrl, {String? deugKey}) :
  _clientId = debugKey == null ? generateUuidV4() : '$debugKey-${generateUuidV4()}' {
```